contrib/checkbox-provider-ce-oem/units/strict-confinement/powermanagement-strict-confinement.pxu

```diff
@@ -5,20 +5,19 @@ _description: This test will check your system shutdown/booting cycle via dbus c
 unit: job
 plugin: shell
 environ: STRESS_BOOT_WAKEUP_DELAY RTC_DEVICE_FILE
-command:
-  set -e
-  rtcwake -d "${RTC_DEVICE_FILE:-rtc0}" -v -m on -s "${STRESS_BOOT_WAKEUP_DELAY:-120}" &
-  test-strict-confinement.reboot cold
 user: root
 flags: preserve-locale noreturn autorestart
 estimated_duration: 180.0
+depends: com.canonical.certification::init-boot-loop-data
 imports:
   from com.canonical.certification import snap
   from com.canonical.certification import lsb
 requires:
   lsb.distributor_id == 'Ubuntu Core'
   snap.name == 'test-strict-confinement'
-depends: com.canonical.certification::init-boot-loop-data
+command:
+  set -e
+  rtcwake -d "${RTC_DEVICE_FILE:-rtc0}" -v -m no -s "${STRESS_BOOT_WAKEUP_DELAY:-120}" && test-strict-confinement.cold-boot || exit 1
```

Comment:
There are two problems with this line:

- This command creates a race against checkbox. As soon as the dbus call is made to Poweroff, Checkbox proceeds to the next job (and writes progress to the disk, which leads to other IO problems, btw.), and if Checkbox starts the next job before the actual poweroff happens, the next job will be interrupted by the system. 

`|| exit 1` is redundant, if the first command failed then the error code will be preserved (also if the second one fails).

Reply:
Hi @kissiel,
Does this race condition happen because the dbus command is called by another SNAP which is outside of the checkbox instead of calling dbus itself? If so, do you have any recommendations for ways to avoid this kind of issue?